base/Dockerfile

```diff
@@ -14,7 +14,8 @@ RUN INSTALL_PKGS=" \
     yum install -y ${INSTALL_PKGS} && \
     rpm -V ${INSTALL_PKGS} && \
     yum clean all && \
-    mkdir -p /var/lib/origin
+    mkdir -p /var/lib/origin && \
+    echo 'skip_missing_names_on_install=0' >> /etc/yum.conf
```

Comment:
Yeah move this up and then nuke `rpm INSTALL_PKGS`

Reply:
sounds good